include/library/spdm_requester_lib.h

```diff
@@ -705,6 +705,31 @@ libspdm_return_t libspdm_set_certificate(void *spdm_context,
                                          const uint32_t *session_id, uint8_t slot_id,
                                          void *cert_chain, size_t cert_chain_size);
 
+/**
+ * This function try to send SET_CERTIFICATE
+ * to set certificate or erase certificate from the device.
+ *
+ * @param  context          A pointer to the SPDM context.
+ * @param  session_id       Indicates if it is a secured message protected via SPDM session.
+ *                          If session_id is NULL, it is a normal message.
+ *                          If session_id is NOT NULL, it is a secured message.
+ * @param  slot_id          The number of slot for the certificate chain.
+ * @param  cert_chain       The pointer for the certificate chain to set.
+ *                          The cert chain is a full SPDM certificate chain, including Length and Root Cert Hash.
+ * @param  cert_chain_size  The size of the certificate chain to set.
+ * @param  request_attribute Set certificate request attributes.
```

Comment:
Please clarify that slot_id of request_attribute in SPDM spec must be 0.

Reply:
Thanks. I have added the comment.